### DISCUSSION TITLE
Error -9 assembly short reads

### DISCUSSION THREAD

**Discussion by Giulica13** [ablab/spades#1300] [Q&A] [answered]
Hi, 
I'm trying to assembly some short reads (around 50-90 bp in lenghts) using SPAdes. For some of my fastq files I obtain this error:

== Error ==  system call for: "['/usr/lib/spades/bin/spades-hammer', '/mnt/17161c71-36f0-4fe6-9a54-4e336ebf44eb/Ls_321R/spades_results_Ls_321R/corrected/configs/config.info']" finished abnormally, err code: -9

Please, can you help me? 
Here my spades.log and params.txt.

[params.txt](https://github.com/ablab/spades/files/15377684/params.txt)
[spades.log](https://github.com/ablab/spades/files/15377685/spades.log)

**Comment by asl** [comment, marked as answer]
Your assembly job was killed. Likely more information could be found from the system log
